shared/vpcs.tf

```diff
@@ -40,6 +18,30 @@ locals {
   experience_cidr_block_vpc     = "172.19.0.0/16"
   experience_cidr_block_public  = "172.19.0.0/17"
   experience_cidr_block_private = "172.19.128.0/17"
+
+  developer_cidr_block_vpc     = "172.42.0.0/16"
+  developer_cidr_block_public  = "172.42.0.0/17"
+  developer_cidr_block_private = "172.42.128.0/17"
```

Comment:
Possibly of interest: Terraform can do this for us.

```
> cidrsubnet("172.42.0.0/16", 1, 0)
172.42.0.0/17

> cidrsubnet("172.42.0.0/16", 1, 1)
172.42.128.0/17
```

https://www.terraform.io/docs/configuration/functions/cidrsubnet.html